src/ROUTES.ts

```diff
@@ -601,9 +601,13 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/members/:accountID/role-selection',
         getRoute: (policyID: string, accountID: number, backTo?: string) => getUrlWithBackToParam(`settings/workspaces/${policyID}/members/${accountID}/role-selection`, backTo),
     },
+    WORKSPACE_TAXES_NEW: {
```

Comment:
@kosmydel @luacmartins NAB, should we update it to `WORKSPACE_TAX_NEW`? I think It is better 

If we decide to update this to `WORKSPACE_TAX_NEW`, please also update all other places for consistency

Reply:
I think we can use here `WORKSPACE_TAX_CREATE` to match others: `WORKSPACE_TAG_CREATE` and `WORKSPACE_CATEGORY_CREATE`.